bin/rector

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env php
-<?php declare(strict_types=1);
+<?php
```

Comment:
Why? :cry:

Reply:
Unfortunatelly it's needed for PHAR to make work.

See PHPUnit:

- https://github.com/sebastianbergmann/phpunit/issues/2791#issuecomment-334953382
- https://github.com/sebastianbergmann/phpunit/commit/70ccf0370f0b99c22819d3730167a7d448911b9e

Reply:
I found a way to revert this :+1: